enro/src/androidTest/AndroidManifest.xml

```diff
@@ -18,5 +18,7 @@
         <activity android:name=".core.ImmediateOpenFragmentChildActivity"/>
         <activity android:name=".result.ViewModelForwardingResultActivity"/>
         <activity android:name=".ActivityWithComposables"/>
+        <activity android:name=".core.ComposableTestActivity" />
+        <activity android:name="androidx.activity.ComponentActivity"/>
```

Comment:
Is this ComponentActivity required if you're only ever launching `ComposableTestActivity`?

Reply:
Yes, because I want to be able to launch the composable rule using `createComposeRule()`, which launches the ComponentActivity to start with. Using `createAndroidComposeRule` which directly launches the Activity under test doesn't appear to work correctly.